MultiPartyExample/LocalParticipantView.swift

```diff
@@ -0,0 +1,148 @@
+//
+
+import UIKit
+import TwilioVideo
+
+@IBDesignable
+class LocalParticipantView: UIView {
+
+    @IBOutlet var contentView: UIView!
+    @IBOutlet weak var videoView: TVIVideoView!
+    @IBOutlet weak var noVideoImage: UIImageView!
+    @IBOutlet weak var audioIndicator: UIImageView!
+    @IBOutlet weak var networkQualityLevelContainerView: UIView!
+    @IBOutlet weak var networkQualityLevelIndicator: UIImageView!
+
+    var recognizerDoubleTap: UITapGestureRecognizer?
+
+    var hasAudio: Bool = false {
+        willSet {
+            audioIndicator.isHidden = newValue
+        }
+    }
+
+    var hasVideo: Bool = false {
+        willSet {
+            videoView.isHidden = !newValue
+            videoView.alpha = newValue ? 1.0 : 0.0
+            noVideoImage.isHidden = newValue
+        }
+    }
+
+    var networkQualityLevel: TVINetworkQualityLevel = .unknown {
+        willSet {
+            let info = networkQualityLevelIndicatorInfo(newValue)
+
+            guard let networkQualityLevelImage = info.networkQualityLevelImage,
+                let tintColor = info.tintColor else {
+                    networkQualityLevelContainerView.isHidden = true
+                    return
+            }
+
+            networkQualityLevelContainerView.isHidden = false
+            networkQualityLevelIndicator.image = networkQualityLevelImage
+            networkQualityLevelIndicator.tintColor = tintColor
+        }
+    }
+
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    private func setup() {
+        Bundle.main.loadNibNamed("LocalParticipantView", owner: self, options: nil)
+        addSubview(contentView)
+        contentView.frame = self.bounds
+        contentView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+
+        audioIndicator.layer.cornerRadius = audioIndicator.bounds.size.width / 2.0;
+        audioIndicator.layer.backgroundColor = UIColor.black.withAlphaComponent(0.75).cgColor
+
+        noVideoImage.isHidden = false
+        networkQualityLevelContainerView.isHidden = true
+        networkQualityLevelContainerView.layer.backgroundColor = UIColor.black.withAlphaComponent(0.75).cgColor
+
+        // `TVIVideoView` supports scaleToFill, scaleAspectFill and scaleAspectFit.
+        // scaleAspectFit is the default mode when you create `TVIVideoView` programmatically.
+        videoView.contentMode = .scaleAspectFit
+        videoView.isHidden = true
+        videoView.alpha = 0.0
+        videoView.delegate = self
+
+        // Double tap to change the content mode.
+        recognizerDoubleTap = UITapGestureRecognizer(target: self, action: #selector(changeLocalVideoAspect))
+        if let recognizerDoubleTap = recognizerDoubleTap {
+            recognizerDoubleTap.numberOfTapsRequired = 2
+            videoView.addGestureRecognizer(recognizerDoubleTap)
+        }
```

Comment:
I am totally fine with keeping this gesture recognizer as it doesn't need to be super obvious / discoverable. 👍